CDCgov/data-exchange-upload
# Upload 787/ps api copy spans

Adds integration with Processing status API for reporting copying to EDAV and Routing storage accounts.  Both operations are captured under one span, but each are reported separately.  This PR only covers the happy path scenario.

Testing steps:
1. Upload a file as you normally would.
2. Query PS API for trace by upload ID
3. Observe a "dex-file-copy" span in the list of spans.
4. Query PS API for report by upload ID
5. Observe two reports under the "dex-file-copy" stage name
